go.mod

```diff
@@ -3,82 +3,117 @@ module github.com/medibloc/panacea-oracle
 go 1.19
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.9
+	github.com/btcsuite/btcd v0.22.1
+	github.com/confio/ics23/go v0.7.0
+	github.com/cosmos/cosmos-sdk v0.45.10
```

Comment:
When I add ibc-go/v2@v2.0.3 as done in doracle, they said that 
```
retracted by module author: depends on SDK version without dragonberry fix
go: to switch to the latest unretracted version, run:
        go get github.com/cosmos/ibc-go/v2@latest
```

So I add the latest version of ibc-go/v2 and it upgrades cosmos-sdk to v0.45.10.
There seems no breaking changes from v0.45.9 to v0.45.10, but I think it would be better to upgrade the version of cosmos-sdk to v0.45.10 in panacea-core as well.

Reply:
Fair point. I'll make a v0.45.10 for `github.com/medibloc/cosmos-sdk`. But, until then, is it not possible to ignore the error that you mentioned? If it's not possible, it's merge this PR as it is.

Reply:
I got it!. We can ignore it so that I change the version of cosmos-sdk to v0.45.9.